app/javascript/components/widget-controller.jsx

```diff
@@ -56,12 +56,18 @@ export const WidgetController = ({ client, cityName }) => {
       variables={{ id: newWidgetId, cityName }}
       fetchPolicy={fetchPolicy}
     >
-      {({ loading, error, data }) => {
+      {/* eslint-disable-next-line no-shadow */}
+      {({ client, loading, error, data, refetch }) => {
```

Comment:
Looks like we disable `no-shadow` here for `client`. Is this the same `client` object that is passed into `WidgetController` above?

Reply:
Yup it's the same object! Maybe I should have listened to the eslint warning 🙃 Figured it was some special Apollo call, but it works fine just reference the parent object.

Pushed new changes to reflect.